packages/tenderly-utils/src/tenderly.ts

```diff
@@ -0,0 +1,126 @@
+import axios from 'axios'
+import { ethers, JsonRpcProvider } from 'ethers'
+import { IAccountGuardAbi, IAccountImplementationAbi } from './abis'
+import { NetworkName, TokenName, tokenAddresses } from './utils'
+import { getTokenDecimals } from './getTokenDecimals'
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const { TENDERLY_USER, TENDERLY_PROJECT, TENDERLY_ACCESS_KEY } = process.env as any
+
+if (!TENDERLY_USER || !TENDERLY_PROJECT || !TENDERLY_ACCESS_KEY) {
+  throw new Error('Tenderly environment variables not set')
+}
+
+const TENDERLY_FORK_API = `https://api.tenderly.co/api/v1/account/${TENDERLY_USER}/project/${TENDERLY_PROJECT}/fork`
+
+const request = axios.create({
+  baseURL: 'https://api.tenderly.co/api/v1',
+  headers: {
+    'X-Access-Key': TENDERLY_ACCESS_KEY,
+    'Content-Type': 'application/json',
+  },
+})
+
+export const createFork = async ({
+  network,
+}: {
+  network: 'mainnet' | 'optimism' | 'arbitrum' | 'base'
```

Comment:
These are exported in `hardhat-utils` as well as the network IDs. Perhaps we can use those types? If not, would you mind to add a TODO to it?

Reply:
If it's fine to have hardhat-utils as dep than yes I can do this :)